modules/displaybook.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-mutable-exports */
```

Comment:
- [ ] Kindly don't disable linters because linters prevents you from making those errors which can break your application 🤔 So please remove the above line and fix your linter errors 😊